contrib.php

```diff
@@ -38,11 +38,10 @@ function multidimsort($array_in, $column)
     reset($multiarray);
```

Comment:
No longer needed with foreach

Reply:
... and the code above looks like another use for array_column. I think you can simplify the entire thing and also get rid of the loopvalue (just append to the array instead).

Reply:
yes you're right